src/core/Modal/ModalContent/ModalContent.tsx

```diff
@@ -17,6 +18,7 @@ interface InternalModalContentProps extends ModalContentProps {
   id?: string;
   modalVariant: ModalVariant;
   scrollable: boolean;
+  theme: SuomifiTheme;
```

Comment:
Could this be done like in some other components, by extending `SuomifiThemeProp` instead of having it told in these props? Or is there certain requirement for it to work correctly to do it like it is now?

E.g
```tsx
class BaseModalContent extends Component<
  InternalModalContentProps & SuomifiThemeProp
> {
```

This same applies to other components that have now `theme` added to their props.

Reply:
No need to have it this way. It was implemented before SuomifiThemeProp was implemented. Will fix.

Reply:
Fixed as proposed